docs/installation.md

```diff
@@ -41,6 +31,14 @@ bin/setup
 
 You can now [clone and setup the apps you need](../README.md#Usage), after which you can do things like run tests and startup the app in your browser. If this doesn't work for whatever reason, follow the [instructions on how to resolve setup issues](#how-tos).
 
+### Docker settings
+
+Running GOV.UK applications can be resource intensive. To give Docker more resources, click the Docker whale icon in the macOS menu bar, select 'Preferences'. We suggest the following minimum resources:
+
+* 6 CPUs
+* 12 GB RAM
+* 64GB+ Disk
+
```

Comment:
Is it worth noting that by default if you run Docker on Linux, this doesn't apply as it can use all the available resources?

Reply:
Sounds reasonable - I've added a caveat.